examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,415 @@
+# simulate the interfacial crack opening. These conditions are applied to the
+# top and bottom nodes corresponding to the geometrical edges located at
+# (0.0, `y`, 0.0) and (0.0, `y`, 3.4) respectively (coordinates (x, y, z)). Two different components
+# are assigned to these sets of nodes for a faster identification of the nodes
+# bearing reaction forces.
+
+# applying the two displacement conditions
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=0.0, vmax=0.0)
+mapdl.nsel(type_="r", item="loc", comp="z", vmin=2*height, vmax=2*height)
+mapdl.d(node="all", lab="uz", value=d)
+mapdl.cm("top_nod", "node")
+
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=0.0, vmax=0.0)
+mapdl.nsel(type_="r", item="loc", comp="z", vmin=0.0, vmax=0.0)
+mapdl.d(node="all", lab="uz", value=-10)
+mapdl.cm("bot_nod", "node")
+
+# applying the fix condition
+mapdl.allsel()
+mapdl.nsel(type_="s", item="loc", comp="x", vmin=length + pre_crack, vmax=length + pre_crack)
+mapdl.d(node="all", lab="ux", value=0.0)
+mapdl.d(node="all", lab="uy", value=0.0)
+mapdl.d(node="all", lab="uz", value=0.0)
+
+mapdl.eplot(
+    # plot_bc=True, bc_glyph_size=3, title="", background="white", show_axes=False
+)
+
+###############################################################################
+# Solve the non-linear static analysis
+# ====================================
+#
+# A non-linear static analysis is run to in this very last step. One hundreds
+# substeps are requested in this case to have a smooth crack opening
+# progression as well as to facilitate convergency for the static solver.
+
+# entering the solution processor and defining the analysis settings
+mapdl.allsel()
+mapdl.finish()
+mapdl.run("/SOLU")
+mapdl.antype("static")
+
+# activating non-linear geometry
+mapdl.nlgeom("on")
+
+# requesting substeps
+mapdl.autots(key="on")
+mapdl.nsubst(nsbstp=100, nsbmx=100, nsbmn=100)
+mapdl.kbc(key=0)
+mapdl.outres("all", "all")
+
+# solving
+output = mapdl.solve()
+
+###############################################################################
+# Post-processing
+# ===============
+# The post-processing is performed using PyMAPDL and PyDPF.
+#
+#
+# Post-processing of the results using PyMAPDL
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This section illustrates how to perform a quick prost-processing of the
+# the results using PyMAPDL. In this particular case as one of the key part is
+# measuring the delamination length, the cohesive damage parameter is plotted.
+# Although, the damage parameter is an element parameter, the result is
+# provided in terms of nodal result. Therefore, in this case we present the
+# result for just one of the four-noded cohesive element i.e. ``NMISC = 70``.
+# The result for the other nodes are present at ``NMISC = 71,72,73``.
+# The actual damage parameter nodal values can also be retrieved from the
+# solved model in form of table (or array).
```

Comment:
```suggestion
# This section shows how to use PyMAPDL to postprocess results. Because
# measuring the delamination length is important, plot the cohesive damage parameter.
# Although the damage parameter is an element parameter, the result is
# provided in terms of a nodal result. Thus, the result for just one of
# the four-noded cohesive element ``NMISC = 70`` is presented.
# The result for the other nodes are present at ``NMISC = 71,72,73``.
# You can retrieve the actual damage parameter nodal values from the
# solved model in form of a table (or an array).
```